src/Root/routes.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Route, IndexRoute } from 'react-router';
+import AuthContainer from '../Auth/AuthContainer';
+import LandingPage from './LandingPage';
+
+export default (
+  <Route path="/" component={AuthContainer}>
+    <IndexRoute component={LandingPage} />
```

Comment:
hmm  I change to use the other way.

https://github.com/esbb48/Kizuna/pull/4#discussion_r74061405